typescript/sdk/src/utils/decimals.ts

```diff
@@ -0,0 +1,31 @@
+export function verifyScale(configMap: Record<string, any>): boolean {
+  if (!areDecimalsUniform(configMap)) {
+    const maxDecimals = Math.max(
+      ...Object.values(configMap).map((config) => config.decimals!),
+    );
+
+    for (const [_, config] of Object.entries(configMap)) {
+      if (config.decimals) {
+        const scale = 10 ** (maxDecimals - config.decimals);
```

Comment:
can this be a little more specific to designate some calculated scale versus config.scale?

```suggestion
        const calculatedScale = 10 ** (maxDecimals - config.decimals);
```